themes/default/content/case-studies/greenpark-sports.md

```diff
@@ -0,0 +1,92 @@
+
+<img class="block mx-auto md:max-w-4xl my-8" src="/images/case-studies/greenpark-sports-deploy-diagram.png">
+<p class="text-sm italic text-center">GreenPark developers deploy environments that typically include a Kubernetes cluster, Confluent Kafka cluster, a MongoDB Atlas database, and Cloudflare.</p>
+
+Jacob had already heard about Pulumi, but a strong recommendation from a former coworker convinced him to take a closer look. He learned that one significant advantage of Pulumi is that it supports general-purpose languages like Go, Python, TypeScript/JavaScript, and C# (.NET). This would enable GreenPark developers to use infrastructure as code with languages and tools they already use on a daily basis, such as Go, Visual Studio, and test frameworks.
+
+Pulumi would also enable Jacob to build, share, and reuse infrastructure code with the full benefits of a software supply chain. His team would be able to build higher-level components that make it easier for developers to build production-quality cloud applications. The components would be wrapper functions and helper functions written in Go, which developers could use to build and deploy a fully-configured Kubernetes application.
+
+When recounting how he first started evaluating Pulumi, Jacob said, “I took a look at Pulumi and that started a much happier time in my life.”
+
+### Putting "Developers First" with Cloud Infrastructure
+
+After deciding on Pulumi, Jacob and his team built a system based on a “developer-first” approach. The system made developers as independent as possible by allowing them to provision cloud infrastructure, with guardrails to maintain security.
+
+#### Migrating Infrastructure to Pulumi
+
+First, the platform team migrated its infrastructure from Terraform to Pulumi. This included Kubernetes resources, Google Cloud resources, GitHub repos, DataDog, CloudFlare, Confluent for Kafka, Consul, and Vault. This enabled them to build, deploy, and manage all of their cloud and SaaS resources from a single platform, using Go as their language of choice.
+
+<img class="block mx-auto md:max-w-4xl my-8" src="/images/case-studies/greenpark-sports-monorepo-diagram.png">
+<p class="text-sm italic text-center">GreenPark Sports uses a monorepo to manage its infrastructure code.</p>
+
+#### Simplifying Kubernetes Deployments
+
+Pulumi also simplified how they manage Kubernetes deployments. It gave them a strongly-typed way of modeling Kubernetes resources with standard languages (e.g., Go) instead of YAML. Using Pulumi’s native support for Kubernetes, they could access 100% of Kubernetes’ APIs. They could also build and manage Helm charts and Kubernetes custom resource definitions with Pulumi’s cloud native integrations.
```

Comment:
```suggestion
Pulumi also simplified how they manage Kubernetes deployments. It gave them a strongly-typed way of modeling Kubernetes resources with standard languages (e.g., Go) instead of YAML. Using Pulumi’s [native support for Kubernetes](/kubernetes), they could access 100% of Kubernetes’ APIs. They could also build and manage Helm charts and Kubernetes custom resource definitions with Pulumi’s cloud native integrations.
```